ui/cryptomaterial/clickable.go

```diff
@@ -60,19 +60,28 @@ func (cl *Clickable) Enabled() bool {
 	return cl.isEnabled
 }
 
-func (cl *Clickable) Layout(gtx C, w layout.Widget) D {
+// LayoutWithInset draws a layout of a clickable and applies an hover effect if
+// an hover action is detected. rightInset and bottomInset are used to restrict
+// hover layout and should be supplied ONLY if a right or bottom inset/margin
+// was applied to w. 
+func (cl *Clickable) LayoutWithInset(gtx C, w layout.Widget, rightInset, bottomInset unit.Dp) D {
 	return cl.button.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
 		return layout.Stack{}.Layout(gtx,
 			layout.Expanded(func(gtx layout.Context) layout.Dimensions {
+				// Only hover on a widget, ignore inset or margin applied.
+				gtx.Constraints.Min.Y = gtx.Constraints.Min.Y - gtx.Dp(bottomInset)
+				gtx.Constraints.Min.X = gtx.Constraints.Min.X - gtx.Dp(rightInset)
```

Comment:
@dreacot, I need to also support accounting for top and left inset so hover effects always stay within expected constraints. Any thoughts?

Reply:
seems to work okay